model.py

```diff
@@ -197,6 +202,17 @@ def initialize(cls, url):
                 view_name, resource_file)
             sql = open(resource_file).read()
             connection.execute(sql)                
+
+        # Create the recursive equivalents function. If the function
+        # already exists, it will be replaced.
+        if not connection:
+            connection = engine.connect()
+        resource_file = os.path.join(resource_path, cls.RECURSIVE_EQUIVALENTS_FUNCTION)
+        if not os.path.exists(resource_file):
+            raise IOError("Could not load recursive equivalents function from %s: file does not exist." % resource_file)
+        sql = open(resource_file).read()
+        connection.execute(sql)
```

Comment:
I _think_ this will run when the app is restarted, so it won't be necessary to have a migration.